templates/case-studies/templates/_macros-hero.jinja

```diff
@@ -0,0 +1,33 @@
+# Params
+# title: Hero title text (required)
+# subtitle: Hero subtitle text
+# logo: Path to logo image
+# logo_alt: Alt text for logo image
+# logo_width: Width of logo image (128 by default)
+# logo_height: Width of logo image (128 by default)
+
+{%- macro hero(
```

Comment:
If possible, it would be good to reuse fully featured Vanilla's hero macro (https://github.com/canonical/vanilla-framework/blob/main/templates/_macros/hero.jinja) so that we start using the dedicated patterns.

I don't know if the hero on case study page matches any of the new approved hero designs (https://staging.vanillaframework.io/docs/patterns/hero), it would be good if it uses one of there.

If not, maybe let's not call this a "hero", but "case_study_hero" to be specific, that this is meant for this particular template only.

Reply:
Looks like you took the markup rendered by the 25/75 signpost hero example, and pasted it into a macro. 

In the interest of keeping our heros as close as possible, maybe it's best to copy the macro we have, rather than its output: https://github.com/canonical/vanilla-framework/blob/main/templates/_macros/hero.jinja (as long as our macro suits your needs, and if it doesn't we should figure out how to adjust so that it does)

Then either call our macro or make a macro that calls our macro

Reply:
Thanks, I have copied the hero macro over since we are using the 25/75 signpost in case studies :)